README.md

```diff
@@ -1,33 +1,174 @@
+You can edit or remove feature cards and the changes will be reflected after the code passes build test. 
+If you are adding or editing an image file, the image file must is present inside `static/img` directory, otherwise the compiler would return an error.
+You should use SVG images when available.
+
+### PEcAn People
+
+The people page contains details of the PEcAn team. The image here is a representation of how the page should look like with placeholder values.
+
+![PecanPople](screenshots/pecanPeople.png)
+
+To add and modify information move to the `static/utils/contributors.js` file. 
+It contains an array called `contributors` which contains JavaScript objects which are enclosed by `{}` and contain following fields-
+
+![Contributors.js](screenshots/contributors.png)
+
+In order to add a contributor, copy any object enclosed by `{}` and paste after the last object. Make sure two objects are separated by a `,`. Now one can easily edit information in the pasted object.
+One could add information accordingly in the different fields. The information would be automatically updated after successful push.
+
+### Blog
+
+A link to blog posts can be found in the navigation menue at the top of the website. 
+The Docusaurus blog feature is described in detail in the [Docusaurus documentation](https://docusaurus.io/docs/blog). 
+By default when someone clicks on `blog` link they will be directed to a page containing list of all present blogs on the website.
+
+Steps to add a blog post:
+
+1. Create a new file named  `blog/yyyy-mm-dd-title.md`
+2. In the top of your blog file add a yaml header.
+3. Add at least `title`, `author` and `tags` yaml fields. 
+  - These metadata fields help with search and navigation on the site. 
+  - These and other yaml header options are described in the [blog header options](https://docusaurus.io/docs/blog#header-options) section of the Docusaurus documentation.   
+4. writing your blog and save the file. 
+5. any images can be put in `static/img/` and referenced using a relative path `static/img/myimage.png` 
+6. build the website to make sure the blog shows up as expected 
+7. After the new changes are merged into the main branch, the new blog post will appear on the website.
+
+![YamlHeader](screenshots/yamlHeader.png)
+
+<!-- instead of that screenshot, perhaps provide a code chunk here  -->
+
+To learn more about markdown visit (https://guides.github.com/features/mastering-markdown/). 
+
+
+## Screenshots
+
+This section contains screenshots of website's homepage in light and dark theme.
+
+![LightTheme](screenshots/light.png)
+
+![DarkTheme](screenshots/dark.png)
+
+## Contributing
+
+This is an open source project and contributions are what make an open source community.
+Everyone is welcome to contribute to this project. The rules are simple:
+
+- Be polite, be patient.
+- Be specific about your doubts, make sure you read this documentation carefully.
```

Comment:
```suggestion
```

just want to stick to the official code of conduct (also to avoid implying that contributors should RTFM before asking any questions)